### ISSUE TITLE
Source control showing nothing again

### ISSUE THREAD

**Issue by craigc40** [microsoft/vscode#90909]
This appears to be the same problem documented in https://github.com/microsoft/vscode/issues/82203, but as of 1.42.1, it is broken again. Workaround is to toggle Scm: Always Show Providers

Version: 1.42.1 (user setup)
Commit: c47d83b293181d9be64f27ff093689e8e7aed054
Date: 2020-02-11T14:45:59.656Z
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.16299

Steps to Reproduce:
I don't know how to reproduce it, because it seems to be working permanently after I applied the above workaround. The problem occurred immediately after I upgraded to the newest version.

**Comment by joaomoreno** [comment]
Can you show me a screenshot of the issue when it comes back?

**Comment by craigc40** [comment]
I tried setting auto-update to manual, downgrading to 1.41.1, checking for updates, and then letting it update to 1.42.1.  That did not reproduce the problem.
Apparently, applying that workaround fixes things. This post from the 82203 bug thread shows what I saw when the bug occurred:
https://github.com/microsoft/vscode/issues/82203#issuecomment-544592959
The 82203 thread is exactly what happened to me. I think it was prematurely closed because the issue is not completely fixed.
